python/mxnet/ndarray/sparse.py

```diff
@@ -1138,6 +1139,8 @@ def _ndarray_cls(handle, writable=True, stype=_STORAGE_TYPE_UNDEFINED):
         stype = _storage_type(handle)
     if stype == _STORAGE_TYPE_DEFAULT:
         return NDArray(handle, writable=writable)
+    elif stype == _STORAGE_TYPE_MKLDNN:
+        return NDArray(handle, writable=writable)
```

Comment:
you are not using MKLNDArray here?

Reply:
right. i wanted to delete MKLNDArray since we don't want to expose MKLDNN to users. But we still want users to specify MKLDNN storage during the inference, so I kept the code as it was. We need to discuss how we allow users to specify MKLDNN storage.